.github/workflows/build.yml

```
@@ -82,7 +82,7 @@ jobs:
         run: |
           cd "${{ matrix.image }}" && sudo chmod a+w . && make update
           if [[ -n "$DOCKER_USERNAME" ]]; then
-            echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin && echo "::set-output name=logged_in::true"
+            echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin && echo "{logged_in}={true}" >> $GITHUB_OUTPUT
```

```suggestion
            echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin && echo "logged_in=true" >> $GITHUB_OUTPUT
```
Looking at the change in the blog, since the brackets weren't there in our before version, so I don't think they should be there in our after version.